src/Pages/Home/components/TopLeftAlertMenu.tsx

```diff
@@ -5,35 +5,38 @@ import { RightArrowSvg } from "../../../components/Icons/RightArrow"
 import { ListOfAlerts } from "./ListOfAlerts"
 import { ArrowTopSvg } from "../../../components/Icons/ArrowTop"
 import { OpenArrowsSvg } from "../../../components/Icons/OpenArrowsSvg"
+import { useTranslation } from "react-i18next"
 
 export function TopLeftAlertMenu() {
+    const {t} = useTranslation();
+    
     const [isAlertOpen, setIsAlertOpen] = useState(false)
+
     if (isAlertOpen) {
-        return <div className="bg-primary text-primary-foreground w-[25vw] relative">
```

Comment:
here do you have duplicate  return and its wrong :
your code had two return statements within the same if block, which isn’t allowed. Consolidating this under a single if block keeps it clean.

please do more simple the code : wrapping the logic under the isAlertOpen condition and then using a separate return block for the default (when isAlertOpen is false) makes the structure more organized and readable

Reply:
Please Inform shaked to don't write like this.
And in the Internal Network i already changed.